packages/eui/src/components/tabs/tabs.tsx

```diff
@@ -67,9 +75,36 @@ export const EuiTabs = forwardRef<EuiTabRef, EuiTabsProps>(
       bottomBorder && styles.bottomBorder,
     ];
 
+    const handleKeyDown = (event: KeyboardEvent) => {
```

Comment:
[microperf] We should memoize this in a useCallback so it's not reinstantiated every rerender
```suggestion
    const handleKeyDown = useCallback((event: KeyboardEvent) => {
```

Reply:
@cee-chen `handleKeyDown` doesn't perform any heavy calculations. Re-rendering a bunch of tabs doesn't seem to expensive either. And memoizing a function comes at a cost. That being said, I'll test the performance in Profiler with and without the useCallback, share the results and we can decide together whether to go for it or skip it. Unless you're strong on this?

Reply:
This could be my PTSD from EuiDataGrid speaking but in general I prefer we memoize all callbacks 🤷 That being said, it's not a blocking request, so feel free to skip it. Profiling it would be excellent if we wanted to go the extra mile, but no worries if it's not worth the time to figure out.